code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/service/impl/ServiceDiscoveryApiServiceImpl.java

```diff
@@ -151,6 +152,19 @@ private String convertToYml(Map<String, Object> dockerComposeData) {
     }
 
     @SuppressWarnings("unchecked")
+    private void excludeLables(Map<String, Object> composeServiceData) {
+        if (composeServiceData.get(InstanceConstants.FIELD_LABELS) != null) {
+            Map<String, String> labels = new HashMap<>();
```

Comment:
Yes, but any particular advantage of doing so? I am not iterating and casting each object of the map.